modules/browser/BrowserCapabilities.js

```diff
@@ -282,8 +282,8 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean} {@code true} if the browser supposrts getDisplayMedia.
      */
     supportsGetDisplayMedia() {
-        return navigator.mediaDevices
-            && navigator.mediaDevices.getDisplayMedia !== undefined;
+        return typeof navigator.getDisplayMedia !== 'undefined'
+            || typeof navigator.mediaDevices.getDisplayMedia !== 'undefined';
     }
```

Comment:
@saghul Should we check here whether navigator.mediaDevices is undefined? We are checking it in supportsDeviceChangeEvent.

Reply:
I don't this it's needed at this point, since all browsers we support have it for sure.